Apps/W1/DynamicsGPHistoricalData/app/src/Pages/HistPayablesApply.Page.al

```diff
@@ -0,0 +1,180 @@
+                }
+                field("Currency Code"; Rec."Currency Code")
+                {
+                    ToolTip = 'Specifies the value of the Currency Code field.';
+                }
+                field("Disc. Taken Amount"; Rec."Disc. Taken Amount")
+                {
+                    ToolTip = 'Specifies the value of the Disc. Taken Amount field.';
+                }
+                field("Write Off Amount"; Rec."Write Off Amount")
+                {
+                    ToolTip = 'Specifies the value of the Write Off Amount field.';
+                }
+                field("Orig. Applied Amount"; Rec."Orig. Applied Amount")
+                {
+                    ToolTip = 'Specifies the value of the Orig. Applied Amount field.';
+                }
+                field("Orig. Discount Taken Amount"; Rec."Orig. Discount Taken Amount")
+                {
+                    ToolTip = 'Specifies the value of the Orig. Discount Taken Amount field.';
+                }
+                field("Orig. Discount Available Taken"; Rec."Orig. Discount Available Taken")
+                {
+                    ToolTip = 'Specifies the value of the Orig. Discount Available Taken field.';
+                }
+                field("Orig. Write Off Amount"; Rec."Orig. Write Off Amount")
+                {
+                    ToolTip = 'Specifies the value of the Orig. Write Off Amount field.';
+                }
+                field("Apply To Post Date"; Rec."Apply To Post Date")
+                {
+                    ToolTip = 'Specifies the value of the Apply To Post Date field.';
+                }
+                field("Apply From Document No."; Rec."Apply From Document No.")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Document No. field.';
+                }
+                field("Apply From GL Posting Date"; Rec."Apply From GL Posting Date")
+                {
+                    ToolTip = 'Specifies the value of the Apply From GL Posting Date field.';
+                }
+                field("Apply From Currency Code"; Rec."Apply From Currency Code")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Currency ID field.';
+                }
+                field("Apply From Apply Amount"; Rec."Apply From Apply Amount")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Apply Amount field.';
+                }
+                field("Apply From Disc. Taken Amount"; Rec."Apply From Disc. Taken Amount")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Discount Taken Amount field.';
+                }
+                field("Apply From Disc. Avail. Taken"; Rec."Apply From Disc. Avail. Taken")
+                {
+                    ToolTip = 'Specifies the value of the Apply From Discount Available Taken field.';
```

Comment:
wrong field value in tooltip? Discount instead of Disc. Other fields use the same abbreviation as the field.